maintainers/maintainer-list.nix

```diff
@@ -9522,6 +9522,12 @@
     githubId = 1866448;
     name = "Eric Bailey";
   };
+  setthemfree = {
```

Comment:
Keep the file sorted alphabetically. S is not among two Y's.

Reply:
Fixed; initially thought of another handle, that's why it ended up between the `y`s.